pandas/core/groupby/groupby.py

```diff
@@ -4887,6 +4888,8 @@ def shift(
 
         fill_value : optional
             The scalar value to use for newly introduced missing values.
+        suffix : str, optional
+            An optional suffix to append when there are multiple periods.
```

Comment:
Does this implementation support multiple periods? If so, can you add tests.

Reply:
You mean specifically for groupby?

Reply:
I decided to not support it for groupby yet if that's ok.

Reply:
For parity it would be great for both the DataFrame and groupby implementations to support these equally

Reply:
To be clear I ended up not implementing shift with multiple periods for groupby. That's why I also cut the suffix option. The suffix doesn't make sense absent the multiple period shift.

Reply:
Sorry if that was unclear.

Reply:
Why not support groupby? It seems like it's the same implementation: a for loop followed by concat. Are there additional difficulties?

Reply:
I ended up adding it, thanks.